tests/e2e/vsan_test.go

```diff
@@ -77,12 +82,24 @@ func (s *VsanTestSuite) TearDownTest(c *C) {
 
 var _ = Suite(&VsanTestSuite{})
 
+func (s *VsanTestSuite) newCName() string {
```

Comment:
method doc or improved method name would be useful to know about the intention behind this helper.